### PR TITLE
Fix allow_any_header setter

### PR DESCRIPTION
**Pull request by Guara92**
## PR Type
Bug Fix

## PR Checklist
Check your PR fulfills the following:

- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made for the appropriate packages.
- [x] Format code with the latest stable rustfmt

## Overview
Fix `allow_any_header` method, setting `allowed_headers` field instead of `allowed_origins`

### CODE REVIEW

File: actix-cors/tests/tests.rs [hunk 1]
```diff
@@ -386,3 +386,24 @@ async fn validate_origin_allows_all_origins() {
...
+        .unwrap();
+
+    let req = TestRequest::with_header(header::ACCESS_CONTROL_REQUEST_METHOD, "POST")
+        .header(header::ACCESS_CONTROL_REQUEST_HEADERS, "content-type")
+        .header(header::ORIGIN, "https://www.example.com")
+        .method(Method::OPTIONS)
+        .to_srv_request();
+
+    let resp = test::call_service(&mut cors, req).await;
+    dbg!(&resp);
```
Comment (robjtede) [inline actix-cors/tests/tests.rs:407]:
```suggestion
```

**Review by robjtede — APPROVED** [review]
good spot. just need to remove that dbg

**Review by JohnTitor — APPROVED** [review]
Oh, I totally missed this in my review, thanks for fixing :)

**Comment by robjtede** [comment]
released as 0.5.1

**Pull request merged**